credsweeper/rules/config.yaml

```diff
@@ -26,7 +26,7 @@
   type: pattern
   values:
   - (?P<value>(AKIA|ASIA)[0-9A-Z]{16})
-  - (?P<value>[0-9a-zA-Z/+]{40})
+  - ([^0-9a-zA-Z/+])(?P<value>[0-9a-zA-Z/+]{40})([^0-9a-zA-Z/+])
```

Comment:
Could you explain why `([^0-9a-zA-Z/+])` strings are added before and after original pattern?
If AWS only use 40 characters to secret, I think original one is better than this.

Reply:
Unfortunately - {40} means that the value contains 40 symbols, but left and right symbols are not involved. They might be various.
As you can see from test sample - it is.
Probably we shall test others patterns in the PR to reduce FP.

Reply:
Okay..
Could you test them by CredData?
There is benchmark function.
Changing rule config needs to prove improvement and not to increase FN.

Reply:
After update here is changes: https://github.com/babenek/CredSweeper/commit/762e6367a53109af4d13e649d449fd950b8a158d
Moreover, i compared results and found - many 'AWS ID' were found in lines which are longer for 1 symbol (e.g. AKIA0123456789ABCDEFG)